File: Serene3/Serene3.Web/Modules/com/payments/paymentsColumns.cs
using Serenity.ComponentModel;
using System;
using System.ComponentModel;

namespace Serene3.com.Columns;

[ColumnsScript("com.payments")]
[BasedOnRow(typeof(paymentsRow), CheckNames = true)]
public class paymentsColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    public string OrderUserid { get; set; }
    [EditLink]
    public string Provider { get; set; }
    public string ProviderPaymentId { get; set; }
    public decimal Amount { get; set; }
    public string Currency { get; set; }
    public string Status { get; set; }
    public string RawResponse { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}

File: Serene3/Serene3.Web/Modules/ServerTypes/com/paymentsColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { paymentsRow } from "./paymentsRow";

export interface paymentsColumns {
    Id: Column<paymentsRow>;
    OrderUserid: Column<paymentsRow>;
    Provider: Column<paymentsRow>;
    ProviderPaymentId: Column<paymentsRow>;
    Amount: Column<paymentsRow>;
    Currency: Column<paymentsRow>;
    Status: Column<paymentsRow>;
    RawResponse: Column<paymentsRow>;
    CreatedAt: Column<paymentsRow>;
    UpdatedAt: Column<paymentsRow>;
}

export class paymentsColumns extends ColumnsBase<paymentsRow> {
    static readonly columnsKey = 'com.payments';
    static readonly Fields = fieldsProxy<paymentsColumns>();
}